### ISSUE TITLE
docs: managing containers

### ISSUE THREAD

**Issue by jtagcat** [overnode-org/overnode#31]
https://overnode.org/docs/overnode-yml-file-description
> id property is an unique within a cluster identifier of a project. Overnode can manage multiple projects in the same cluster and the id field value allows to identify a project.

although it is linked to, the fact that multiple 'projects' (the concept is never explained) can be shall be mentioned. I got the impression that the id meant 'cluster name'

**Comment by avkonst** [comment]
Agree. Multiple projects concept should be explained in the workflow essentials.
